scripts/e2etests/VSUtils.ps1

```diff
@@ -55,11 +55,20 @@ function GetVSIDEFolderPath
 
 function KillRunningInstancesOfVS
 {
-    $processName = 'devenv'
-    Write-Host "Kill any running instances of $processName..."
-    (Get-Process "$processName" -ErrorAction SilentlyContinue) | Kill -ErrorAction SilentlyContinue
-
-    WaitForProcessExit -ProcessName "$processName" -TimeoutInSeconds 30
+    Get-Process | ForEach-Object {
```

Comment:
Pretty sure you can simplify all this logic by doing:

Get-Process | Where-Object {$_.Path -like "C:\Program Files (x86)\Microsoft Visual Studio"} | Stop-Process -EA SilentlyContinue -Force

Reply:
that was my first attempt but -like doesn't behave like i would expect it to so i stopped fighting powershell

Reply:
Or just do 1 pipe if you want to log which processing you are trying to kill explicitly.. (although that stop process might log on it's own)

Reply:
there is only 1 pipe here, not sure what you are trying to say

Reply:
My example has 2 piples...you can still replace like with the C# comparison though.

Reply:
is there any additional benefit of going that route?

Reply:
Fewer lines, but still readable :)
Writing powershell, rather than C# when appropriate :)

Reply:
if it blows up, no way to debug just looking at logs , so i'd go with more lines here :)

Reply:
Sounds like you don't trust powershell :D 
I'm suggesting do it, if you can get the same amount of logging :)

In the end, it's not production code, so not worried about it either way personally :)